src/StdString.h

```diff
@@ -252,7 +252,7 @@ inline int ssicmp(const CT* pA1, const CT* pA2)
 
 inline void sslwr(char *pT, size_t nLen)
 {
-	MakeLower( pT, nLen );
+	//MakeLower( pT, nLen );
```

Comment:
You commented out one `MakeLower`, but not the other or the `MakeUpper` equivalents? Did I miss some?